# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1243632728 14400
#      Fri May 29 17:32:08 2009 -0400
# Node ID 38ad8c99d6a2ddcf4db4ccc854cdd54a1606969f
# Parent  2ed8d2d92507c09791fddc491588846e72932283
__gnuplot_get_var__.m: insert missing semicolon

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2009-05-29  John W. Eaton  <jwe@octave.org>
+
+	* plot/__gnuplot_get_var__.m: Insert missing semicolon.
+
 2009-05-28  Ben Abbott <bpabbott@mac.com>
 
 	* plot/__go_draw_axes__.m: Set x2range when xaxislocation=='top' and set
 	y2range when yaxislocation=='right'. Simplified support for ticklabel
 	separator '|'.
 	* plot/__go_draw_figure__.m: Change 'autoscale fix' to 'autoscale keepfix'.
 	* plot/colorbar.m: Add demos.
 
diff --git a/scripts/plot/__gnuplot_get_var__.m b/scripts/plot/__gnuplot_get_var__.m
--- a/scripts/plot/__gnuplot_get_var__.m
+++ b/scripts/plot/__gnuplot_get_var__.m
@@ -123,17 +123,17 @@ function gp_var_value = __gnuplot_get_va
       ## Direct gnuplot to print to <STDERR>
       fputs (ostream, "set print;\n");
       fflush (ostream);
 
       str = {};
       while (isempty (str))
         str = char (fread (istream)');
         if (! isempty (str))
-          str = regexp (str, "OCTAVE:.*", "match")
+          str = regexp (str, "OCTAVE:.*", "match");
           str = str{end}(8:end);
         endif
         fclear (istream);
       endwhile
     endif
 
     ## Strip out EOLs and the continuation character "|"
     str(str=="\n") = "";
